spec/tests/volumecontroller/volumecontroller_spec.rb

```diff
@@ -17,8 +17,13 @@
 #
 
 # when ceph is enabled, ensure we have the ceph keyring installed
-describe command("bash -c 'if grep -q volume_backend_name=ceph /etc/cinder/cinder.conf; then test -f /etc/ceph/ceph.client.cinder.keyring; fi'") do
-  it { should return_exit_status 0 }
+rbd_enable = file('/etc/cinder/cinder.conf').contain("volume_driver=cinder.volume.drivers.rbd.RBDDriver",nil,nil)
```

Comment:
As you did for glance, could you update your check to match start of line and eventual whitespaces (something like `^volume_driver\s+=\s+cinder…`) ?